tests/test_sanitize.py

```diff
@@ -25,6 +25,7 @@ def test_escape_html(self):
     def test_clean_tag(self):
         assert_equal(
```

Comment:
This assertion is incorrect; the signature for `assert_equal` is `assert_equal(obj1, obj2, message)`. So you would have to do something like:

```python
result = sanitize.clean_tag(...)
assert_equal(result, '&39...')
```

Are the values `"&quot;..."` relevant anymore?